syntax/blame.sublime-settings

```diff
@@ -0,0 +1,16 @@
+{
+	"always_show_minimap_viewport": false,
+	"auto_indent": false,
+	"detect_indentation": false,
+	"draw_indent_guides": false,
+	"draw_white_space": "None",
+	"font_options": [],
```

Comment:
Docs ses 
```
// Valid options are "no_bold", "no_italic", "no_antialias", "gray_antialias",
// "subpixel_antialias", "no_round" (OS X only), "gdi" (Windows only) and
// "directwrite" (Windows only)
 ```
But why do we want to abonded the user generall setting ? 
To me honest I not sure I understand the setting. Maybe I just dump.  Just explain why we want to set this?

Reply:
This setting targets "no_bold", "no_italic" to be disabled. But you are right, disabling everything might disturb.